models/AllArticlesmodel.js

```diff
@@ -0,0 +1,7 @@
+const connection = require("../db/connection");
+exports.selectAllArticles = () => {
+  return connection.query(`SELECT * FROM articles`).then((result) => {
```

Comment:
Here you will have to build an aggregate function to retrieve the comment count 😨